src/test/java/com/example/universe/simulator/entityservice/unit/services/PlanetServiceTest.java

```diff
@@ -0,0 +1,141 @@
+class PlanetServiceTest {
+
+    @Mock
+    private PlanetRepository repo;
+
+    @InjectMocks
+    private PlanetService service;
+
+    private static UUID id;
+    private static Planet planet;
+    private static List<Planet> planets;
+
+    @BeforeAll
+    static void setUpContext() {
+        id = UUID.randomUUID();
+        planet = Planet.builder().id(id).name("name").version(0L).build();
+        planets = List.of(planet);
+    }
+
+    @Test
+    @DisplayName("test planetService.get success")
+    void testGet() throws AppException {
+        // arrange
+        given(repo.findById(id)).willReturn(Optional.of(planet));
+        // act
+        Planet resultPlanet = service.get(id);
+        // assert
+        assertThat(resultPlanet).isEqualTo(planet);
+        then(repo).should().findById(id);
+    }
+
+    @Test
+    @DisplayName("test planetService.get fail")
+    void testGet_entityNotFound() {
+        // arrange
+        given(repo.findById(id)).willReturn(Optional.empty());
+        // act, assert
+        AppException appException = catchThrowableOfType(() -> service.get(id), AppException.class);
+        assertThat(appException.getErrorCode()).isEqualTo(ErrorCodeType.ENTITY_NOT_FOUND);
+        then(repo).should().findById(id);
+    }
+
+    @Test
+    @DisplayName("test planetService.getList")
+    void testGetList() {
+        // arrange
+        given(repo.findAll()).willReturn(planets);
+        // act
+        List<Planet> resultPlanets = service.getList();
+        // assert
+        assertThat(resultPlanets).isEqualTo(planets);
+        then(repo).should().findAll();
+    }
+
+    @Test
+    @DisplayName("test planetService.add")
+    void testAdd() {
+        // arrange
+        given(repo.save(planet)).willReturn(planet);
+        // act
+        Planet resultPlanet = service.add(planet);
+        // assert
+        assertThat(resultPlanet).isEqualTo(planet);
+        then(repo).should().save(planet);
+    }
+
+    @Test
+    @DisplayName("test planetService.update success")
+    void testUpdate() throws AppException {
+        // arrange
+        given(repo.findById(id)).willReturn(Optional.of(planet));
+        given(repo.save(planet)).willReturn(planet);
+        // act
+        Planet resultPlanet = service.update(planet);
+        // assert
+        assertThat(resultPlanet).isEqualTo(planet);
+        then(repo).should().findById(id);
+        then(repo).should().save(planet);
+    }
+
+    @Test
+    @DisplayName("test planetService.update fail")
+    void testUpdate_entityNotFound() {
+        // arrange
+        given(repo.findById(id)).willReturn(Optional.empty());
+        // act, assert
+        AppException appException = catchThrowableOfType(() -> service.update(planet), AppException.class);
+        assertThat(appException.getErrorCode()).isEqualTo(ErrorCodeType.ENTITY_NOT_FOUND);
+        then(repo).should().findById(id);
+        then(repo).should(never()).save(planet);
```

Comment:
Don't use specific parameters in method call checks when `never()` is used. Use `ArgumentMatchers`, e.g. `any()`. In your case: `then(repo).should(never()).save(any())`.